i18n/qgis_fr.ts

```diff
@@ -66496,12 +66509,12 @@ et re-chiffrée en utilisant le nouveau mot de passe</translation>
     <message>
         <location filename="../src/ui/mesh/qgsmeshrenderervectorsettingswidgetbase.ui" line="215"/>
         <source>Scaled to Magnitude</source>
-        <translation type="unfinished"></translation>
+        <translation>Adapté à la magnitude</translation>
     </message>
     <message>
         <location filename="../src/ui/mesh/qgsmeshrenderervectorsettingswidgetbase.ui" line="220"/>
         <source>Fixed</source>
-        <translation>Fixé</translation>
+        <translation>Fixe</translation>
```

Comment:
I'm not sure: `Fixe` -> `Fixé`

Reply:
I also hesitated but I think it might be worth giving a look to the context...
![image](https://user-images.githubusercontent.com/7983394/52850428-c2920580-3113-11e9-8dee-7f472b90191b.png)

Reply:
Btw, quelqu'un a une idée de la traduction pour "% of shaft length" (moi connais pas format mesh et son vocabulaire)?

Reply:
Indeed, it's tendentious... I clearly don't have a strong opinion :).

Reply:
I think in this case this is used to determine the proportion of the head of the vector/arrow in relation to the line of the body. Translation could be something like "% de la longueur". Though I'm not sure if the shaft of the arrow is counted from the tip of the vector of from the end of the arrow end. Translation would change depending on the case.